assets/pulpcore-api.prep

```diff
@@ -1,11 +1,20 @@
 #!/usr/bin/execlineb -P
 
-if -n {
-  ls /etc/pulp/certs
-}
 foreground {
+  if -n {
+    ls /etc/pulp/certs
+  }
   mkdir -p /etc/pulp/certs
 }
+foreground {
+  if -n {
+    ls /etc/pulp/certs/database_fields.symmetric.key
+  }
+  openssl rand -base64 32 > /etc/pulp/certs/database_fields.symmetric.key
+}
+if -n {
+  ls /etc/pulp/certs/token_private_key.pem
+}
 foreground {
   openssl ecparam -genkey -name prime256v1 -noout -out /etc/pulp/certs/token_private_key.pem
 }
```

Comment:
The line to generate the public key should also be part of this block.

I'm concluding it's only a public key and not a certificate, but it's still not a good practice to regenerate an identical key over and over.

Reply:
It is part of the "block" due to the weird syntax of execline.
The `if` in line 15 is either exiting the script immediately, or tail call into the rest.

Don't confuse the `{}` with block structure here.

Reply:
Wow. thank you!